documentation/hail_batch.md

```diff
@@ -0,0 +1,75 @@
+# Hail Batch methods
+
+The primary execution environment for CPG code is cloud infrastructure, with workflows
+scheduled using Hail Batch. This collection of methods is designed to make interactions
+with batches, environments, and authentication protocols simpler.
+
+The `init_batch` method is a wrapper for hail.init_batch, used to start a hail query batch
+using the correct billing project and remote temporary directory sourced from the execution
+environment.
+
+```python
+from cpg_utils.hail_batch import init_batch
+
+
+def main():
+    # initiate a hail query back end
+    init_batch()
+
+    # hail object methods
+    ...
+```
+
+When running a Hail Batch workflow using the analysis-runner, the core Driver image will
+have authentication set up, but separate child batches will start with fresh containers.
```

Comment:
```suggestion
have authentication set up, but separate child batches will start with fresh jobs.
```

Reply:
I don't think either of these is the exact right message. The message I intended was that although the Driver container has authentication, the same auth also needs to be triggered within containers used for each batch job. E.g. you can run a `gsutil cp` in the driver image, but shouldn't expect that to work in a job container, unless you trigger auth there as well.

Reply:
Yes, I think that's clear from the current text, thanks!